go.mod

```
@@ -1,47 +1,47 @@
 module k8s.io/cloud-provider-gcp
 
-go 1.21
+go 1.22.2
```

```suggestion
go 1.22.0
```
For `gardener/gardener` we decided to stick on the `.0` version in order to escape the Go toolchain update hell 😅